cluster/manifests/psp/pod_security_policy.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if eq .Cluster.ConfigItems.pod_security_policy_enabled "true" }}
```

Comment:
I don't understand this. The Pull Request disables by default PSPs via admission controller. But this change does _not_ apply by default this manifest. Can you clarify?

Reply:
From the PR description:

---- 

I have set:

```
teapot_admission_controller_pod_security_policy_enabled=false
pod_security_policy_enabled=true
```

In all clusters so merging this will not enable the feature, but we can do it per cluster/channel.

----

So the idea was to remove the config-items per cluster in order to activate this (less cleanup afterwards).

If this is too confusing then I can also flip the logic to set the config-items to enable the new feature.

Reply:
What it confuses me is the default of `pod_security_policy_enabled` is `false`: https://github.com/zalando-incubator/kubernetes-on-aws/pull/6732/files#diff-d2b7fb562825d0d0a4261e91eb9f214a1fefbc247b40666e0b976213473389c5R626

Reply:
I think it's fine either way, i.e. the logic is correct: If `pod_security_policy_enabled: true` the PSP manifests _are_ applied and _not_ deleted, whereas when `pod_security_policy_enabled: false` the PSP manifest _are not_ applied and _are_ deleted.

We just need to remember to _delete_ the config item to enable this feature, rather than _set_ it.